### PR TITLE
Added trove classifiers (#41)

### PR DESCRIPTION
**Pull request by kavdev**
Definitely double check against [this list](https://pypi.python.org/pypi?%3Aaction=list_classifiers) and let me know if you want anything added/removed.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from setuptools import setup, find_packages
+,from setuptools import setup, find_packages
```
Comment (elbuo8) [inline setup.py]: ?

**Comment by kevin-brown** [comment]
This should probably include the general "Python 2" and "Python 3" classifiers, which are used by a few projects to determine if a package has Python 2/3 support.

```
Programming Language :: Python :: 2
Programming Language :: Python :: 3
```

File: setup.py (hunk 1 above)
Comment (kavdev) [inline setup.py]: typo from using the github editor. fixing now.

File: setup.py [hunk 2]
```diff
@@ -15,4 +15,20 @@
...
+        "Framework :: Django".
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: MIT License",
+        "Natural Language :: English",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python :: 2",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.4",
+        "Programming Language :: Python :: 3.5",
```
Comment (andriisoldatenko) [inline setup.py]: Please add
```
        'Programming Language :: Python',
        'Programming Language :: Python :: 3.6',
```

**Comment by andriisoldatenko** [comment]
@kavdev Can you please rebase from master last changes?

**Comment by kavdev** [comment]
@andriisoldatenko: I believe you can do that now with the merge dropdown here on GitHub

**Comment by andriisoldatenko** [comment]
LGTM
@kavdev Thanks a lot!

**Pull request merged**